index.d.ts

```diff
@@ -3,6 +3,7 @@ declare module 'react-native-matomo-sdk' {
     export function initialize(apiUrl: string, siteId: number): Promise<void>;
     export function trackView(route: string[]): Promise<void>;
     export function trackEvent(category: string, action: string, name?: string, value?: number): Promise<void>;
+    export function setUserId(userId: string): void;
```

Comment:
I think it would be useful if the function returned a promise to let the caller know when the userId was successfully set.